test/browser/focus.test.js

```diff
@@ -54,7 +54,7 @@ describe('focus', () => {
 		teardown(scratch);
 	});
 
-	it('should maintain focus when swapping elements', () => {
+	it.skip('should maintain focus when swapping elements', () => {
```

Comment:
should this be skipped?

Reply:
Yes, Swapping in react: https://codesandbox.io/s/olq48z68r5 this is not supported in most frameworks and can lead to unwanted behavior imo

Reply:
@JoviDeCroock The linked sandbox just appends new `h1` elements for me

Reply:
It swaps the input from before to after the h1 and that loses focus on the input or am I misunderstanding it? @marvinhagemeister

Reply:
wait am I missing something there? what does it swap?

Reply:
it swaps the input to bbefore the H1 same behavior as the test that is skipped?

Reply:
The h1 stays?

Reply:
Maybe you forgot to save the sandbox. Re-created it where the elements are swapped all the time: https://codesandbox.io/s/0yo8kxqjzn . Can confirm that `react` loses focus there, too 👍